torch/csrc/jit/codegen/cuda/runtime/helpers.cu

```diff
@@ -28,19 +28,19 @@ __device__ constexpr int64_t ceilDiv(int a, int64_t b) {
 }
 
 __device__ constexpr int max(int a, int b) {
-  return ::max(a, b);
+  return a > b ? a : b;
```

Comment:
Do you see any difference in performance?

Reply:
That's a good point, I should check. The only reason I wanted to change these functions is although nvrtc doesn't complain nvcc does complain. If you have a better idea to make this compliable with nvcc I'm not particularly attached to defining it like this.